src/widgets/playlist/PlaylistInfo.tsx

```
@@ -4,14 +4,14 @@ import styled from 'styled-components'
 
 import { usePlaylist } from '@/app/providers/PlayerProvider'
 import { Cancel } from '@/assets/icons'
-import type { PlaylistDetailResponse } from '@/entities/playlist'
+import type { PlaylistDetail } from '@/entities/playlist'
 import { getGenreLabel } from '@/shared/lib'
 import { flexColCenter } from '@/shared/styles/mixins'
 import { Error, Header, Link, Loading, SvgButton } from '@/shared/ui'
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)